test/unit/attributes/initializer.spec.js

```diff
@@ -1,9 +1,37 @@
 const { expect } = require('chai');
+const { spy } = require('sinon');
+const { attributes } = require('../../../src');
 const Initializer = require('../../../src/attributes/initializer');
 
 describe('Initializer', () => {
   let attributeDescriptor, instance;
 
+  describe('.initialize', () => {
+    let User, userInstance, schema, attributesSpy;
+
+    beforeEach(() => {
+      User = attributes({ name: String })(class User {});
+      userInstance = new User;
+      schema = { name: String };
+
+      attributesSpy = spy(userInstance, 'attributes', ['set']);
+    });
+
+    afterEach(() => attributesSpy.set.resetHistory());
```

Comment:
I think since we create a new instance and a new spy before each test we won't need it.